recipes/dockerspawner/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "dockerspawner" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ac7a8275dd33a73e31da9d3d13849da579f0f248377d5fa558d1ceb793190a6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Comment:
this likely needs to have --no-deps in it

Reply:
yep - found that. Why did it change?

Doesn't it also need `--ignored-installed` and `--no-cache-dir`?